notes/lecture25.tex

```diff
@@ -0,0 +1,180 @@
+
+
+\end{definition}
+\subsection{Linear Programming}
+We'll now adopt the logarithmic barrier method to solve the linear programming (LP) problem defined as follows:
+\begin{equation}
+\label{eq:LP_def}
+\text{LP}:\qquad
+\begin{aligned}[c|c|c]
+&\min_x &c^T&x\\
+&\qquad\text{s.t.} &Ax&\ge b
+\end{aligned}
+\end{equation}
+where $A\in\mathbb{R}^{m\times n}$ with $m\ge n$ and rank($A$)=n. Denote $x^*$ as the optimal point. 
+
+First we write out the augmented cost function by the logarithmic barrier method, i.e.,
+\begin{equation}
+\label{eq:LP_aug_cost}
+f_\epsilon(x) = c^T x-\epsilon \sum_{j=1}^m\ln\left(A^T_j x-b\right).
+\end{equation}
+where $A^T_j$ is the $j$-th row of $A$. Define $x^*_\epsilon=\argmin_x f_\epsilon(x)$. 
+
+\noindent\textbf{Fact}: $x^*_\epsilon$ exists and is unique for any $\epsilon>0$.
+\begin{proof}
+We can easily check that $f_\epsilon(x)$ is convex (as a sum of two convex functions). Therefore, the minimizer $x^*_\epsilon$ must exist and is unique.
+
+\textit{Remark}: For the convexity of $f_\epsilon$, we can also check the second-order derivative, which is positive definite as shown in (\ref{eq:hessian_f_eps}) later.
+\end{proof}
+
+\subsubsection{Central Path}
+The central path of the LP problem in \ref{eq:LP_def} is depicted by the set of $\left\{x^*_\epsilon|\epsilon>0\right\}$, as shown in Fig.~\ref{fig:central_path}.
+\begin{figure}[h!]
+\centering
+\begin{tikzpicture}
+\draw[gray, thick] (-1,2) -- (2,2) -- (3,-1) -- (0.5,-2) -- (-2,-1) -- cycle;
+\filldraw[black] (0,0) circle (2pt) node[anchor=west] {$x^*_\infty$};
+\filldraw[red] (0.5,-2) circle (2pt) node[anchor=north] {$x^*$};
+\draw[blue, thin] (0,0) .. controls (-0.5,-1) .. (0.5,-2);
+\draw[red, thin] (0,0) -- (-0.5,-0.5) -- (-0.1,-0.8) -- (-0.4,-1.1) -- (0.05,-1.4) -- (-0.05,-1.6) -- (0.5,-2);
+\end{tikzpicture}
+\caption{The central path\label{fig:central_path}}
+\end{figure}
+
+Our goal is to design an algorithm that will approximately follow the central path. Assume that we already get a "good" enough initial point, then at every step, we apply one step of Newton's method. To guarantee that the algorithm converges, we need to answer the following two questions:
+\begin{itemize}
+\item Under what conditions that the single-step Newton method works?
+\item How to carefully update $\epsilon$?
+\end{itemize}
+
+\subsubsection{Newton Decrement}
+To apply the Newton's method, first we need to find out the first-order and second-order derivatives of $f_\epsilon$. Note that
+\begin{eqnarray}
+\triangledown f_\epsilon(x)&=&c-\epsilon \sum_{j=1}^m \dfrac{A_j}{A^T_j x-b}\triangleq c-\epsilon A^T S^{-1}\mathbb{1}\label{eq:grad_f_eps}\\
+\triangledown^2f_\epsilon(x)&=&\epsilon A^TS^{-2}A=\epsilon \sum_{j=1}^m\dfrac{A_j A_j^T}{S_j^2}\label{eq:hessian_f_eps}
+\end{eqnarray}
+where $\mathbb{1}=[1,1,\cdots,1]^T\in\mathbb{R}^{m\times1}$, $S=diag\{A^T_1 x-b, A^T_2 x-b, \cdots, A^T_m x-b\}$.
+
+Then the Newton's update can be applied:
```

Comment:
either write "Newton's update" without an article or "the Newton update", but not "the Newton's update"